keycloakapi/src/main/kotlin/de/klg71/keycloakmigration/keycloakapi/KeycloakClientHelper.kt

```diff
@@ -180,12 +180,12 @@ fun Response.extractLocationUUID(): UUID {
 }
 
 fun KeycloakClient.realmById(id: String) =
-    realms().firstOrNull { it.id == id } ?: throw KeycloakApiException("Realm with id: $id does not exist!")
```

Comment:
I guess we can add this "or"-condition here but we should add a comment in my eyes stating why we did it.

Reply:
added comment